### PR TITLE
Stop redefining the same utility function everywhere

### PR DESCRIPTION
**Pull request by Shaptic**
People keep copy-pasting the helper `yeetTx` and it makes it hard to update the implementation.

### CODE REVIEW

**Comment by JFWooten4** [comment]
Re style guide, how do we feel about the [diction](https://www.today.com/parents/teens/yeet-slang-meaning-rcna163660) here @Shaptic?

File: docs/build/guides/transactions/submit-transaction-wait-js.mdx [hunk 1]
```diff
@@ -12,14 +12,15 @@ import {
   FeeBumpTransaction,
   SorobanRpc,
 } from "@stellar/stellar-sdk";
+import { Server, Api } from "@stellar/stellar-sdk";
```
Comment (ElliotFriend) [inline docs/build/guides/transactions/submit-transaction-wait-js.mdx:15]:
```suggestion
import { Server, Api } from "@stellar/stellar-sdk/rpc";
```

I think we want the `/rpc` at the end, right? (legit asking, i'm not entirely sure lol)

**Review by ElliotFriend — COMMENTED** [review]
this is amazing! Thank you for tackling this problem! I know for sure people have over-copy/pasted this, and I've wanted to comb through and get rid of the cruft for a while! 🙏🏻

File: docs/build/guides/transactions/submit-transaction-wait-js.mdx (hunk 1 above)
Comment (Shaptic) [reply docs/build/guides/transactions/submit-transaction-wait-js.mdx:15]: Yep! Nice catch

**Comment by Shaptic** [comment]
@JFWooten4 I mean it's leftover as an informal name from an age-old quick reference lol (yeet as in throw), but you're right that it probably doesn't belong as the name for proper documentation.

**Comment by JFWooten4** [comment]
Frankly, I don't know the exactly legacy contexts, but this sort of context is super helpful in drafting community contribution standards, be the ideas explicit in writing or implicit through example(s). 🫱🏻‍🫲🏽 Appreciate your perspective, @Shaptic! You are the reason my dreams revolve around getting Microsoft to allow 🧠 reactions to comments

**Pull request merged**